test/fixtures/sidecar-containers/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
Given the `kind: Deployment`, shouldn't apiVersion be apps/v1? https://github.com/snyk/kubernetes-monitor/blob/staging/test/fixtures/java-deployment.yaml#L1-L2

Reply:
Probably https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment

Why is the `extensions/v1beta1` here first? Is it part of Deployment kind too?